VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -154,9 +154,13 @@ def exit_if_same_seq(self, exit_status=None):
             sys.exit(0)
 
     def log(self, message):
+        # write message to stderr for inclusion in QOS telemetry
+        sys.stderr.write(message)
```

Comment:
> sys.stderr.write(message) [](start = 7, length = 26)

do these get captured in the pivoted root state as well? Would be great to know how to view these logs on the backend - Kusto?

Reply:
Guest agent monitors stderr.  Prior to this change no output was written to stderr.   If the process is running under guest agent as parent process, stderr will be captured, reported to host agent, and included in telemetry.  If the process is started as a daemon and running outside of guest agent the output may not fall within this capture chain.